### ISSUE TITLE
Where are the Pony communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/pony#59]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Pony supported by one or more large organizations?
* Does Pony have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Pony? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Pony? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Pony? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Pony?
* Are there popular newsletters for Pony?
* Is Pony taught at programming bootcamps? (If so, what are some examples?)
* Is Pony taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Pony?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by mfelsche** [comment]
Let me try to answer what i can:

* As far as I know there is no large organization supporting Pony. There is nonetheless http://www.wallaroolabs.com/ which is supporting Pony a lot.
* There is no official community manager, Pony is a volunteer project.
* Pony enthusiasts meet on IRC, freenode in the #ponylang channel. 
* There is also the user mailing list: https://pony.groups.io/g/user/
* The closest you get to a popular newsletter is the Last Week in Pony series on ponylang.org: https://www.ponylang.org/categories/last-week-in-pony
* Pony has been developed to great extend at Imperial College London, but i don't know if it is still teached there.

**Comment by kytrinyx** [comment]
This is all super helpful, thank you!

> The whole pony core team acts as community managers.

❤️ 💛 💚 💙 💜 🖤 

> Imperial College London

This is probably a bit like saying "hey, you're from New Zealand, do you know John Smith?"... but do you happen to know Robert Chatley? I *think* I visited him at Imperial College...

**Comment by kytrinyx** [comment]
I've got what I need for now; closing this out. Thank you! 🌼
